src/docs/product/data-management-settings/attachment-scrubbing.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: "Attachment Scrubbing"
+sidebar_order: 4
+keywords: ["pii", "gdpr", "personally identifiable data", "compliance", "minidump"]
+---
+
+
+Events can include attachments. In some cases, attachments may contain PII. [Minidumps](/platforms/native/guides/minidumps) are a common attachment type for native crash reports, and minidumps may contain PII in various sections of the file.
+
+
+## Data Scrubbing Methods in Attachments
+
+When scrubbing attachments, and especially minidump, it is often important to do so without destroying the original file format.
```

Comment:
Suggest:
Scrubbing attachments, and especially minidumps, should be done carefully and without destroying the original file format. Sentry provides some built-in knowledge of some file formats and provides custom [source-selectors](#attachment-source-selectors) to help you write scrubbing rules that won't damage the file when used carefully.